### ISSUE TITLE
Overfitting problems

### ISSUE THREAD

**Issue by Jihao-Li** [AnTao97/dgcnn.pytorch#34]
Thank you very much for your code.

When I utilized your DGCNN model, I encountered overfitting problems. The mF1 metric in training set can reach nearly 90%. However, this metric is only about 40% in test set.  `weight decay` and `dropout` do not work.  This phenomenon is also the case for multiple runs. Could you can help me to solve it? Thanks in advance!

**Comment by antao97** [comment]
Sorry, I didn't encounter this problem before, and no one else told me about this problem. Maybe it's because you run the code incorrectly.

**Comment by jeannotes** [comment]
@JihaoLee did you train from sem seg?
@AnTao97 I train this model, and every area I get 80% iou results, why is that? I didn't change anything

**Comment by Jihao-Li** [comment]
@jeannotes  Yes, I used DGCNN_semseg model.

**Comment by jeannotes** [comment]
@JihaoLee I get >80% iou in every area, did you find out why?

**Comment by Jihao-Li** [comment]
@jeannotes  Which dataset did you use? S3DIS?

**Comment by jeannotes** [comment]
@JihaoLee yes, s3dis dataset

**Comment by Jihao-Li** [comment]
Sorry, I did not use this dataset. I used outdoor scene 3D dataset.

**Comment by antao97** [comment]
@jeannotes I think you didn't change `--exp_name` for each area.
